frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx

```diff
@@ -96,6 +120,13 @@ export const IndividualResponsePage = (): JSX.Element => {
       />
     )
 
+  const responseLinkWithKey = `${
+    window.location.origin
+  }/${getMultirespondentSubmissionEditPath(
```

Comment:
can we include the key in `getMultirespondentSubmissionEditPath`?

Reply:
74b51a9c

Reply:
74b51a9c